backend/src/lib/api-docs/constants.ts

```diff
@@ -42,6 +42,13 @@ export const IDENTITIES = {
   },
   DELETE: {
     identityId: "The ID of the identity to delete."
+  },
+  GET_BY_ID: {
```

Comment:
I'd probably just call this `GET` to be inline with our naming conventions

Reply:
The issue here is we cannot comment by method. There would be more `GET` like Get by slug etc.

So get by id to more specific